code/online_challenges/src/project_euler/problem_042/problem_042.py

```diff
@@ -0,0 +1,1813 @@
+import itertools
+
+
+def compute():
+    ans = 0
+    for s in WORDS:
+        x = sum((ord(c) - ord("A") + 1) for c in s)
+        if is_triangular_number(x):
+            ans += 1
+    return str(ans)
+
+
+def is_triangular_number(n):
+    temp = 0
+    for i in itertools.count():
+        temp += i
+        if n == temp:
+            return True
+        elif n < temp:
+            return False
+
+
+WORDS = [
```

Comment:
Please read this data from a separate file.

Reply:
@arnavb Done.